components/fxa-client/android/build.gradle

```diff
@@ -145,7 +145,15 @@ afterEvaluate {
             productFlavor += "${it.name.capitalize()}"
         }
         def buildType = "${variant.buildType.name.capitalize()}"
-        if (variant.buildType.name != 'withoutLib') {
+        // The "CI" check is a terrible hack:
```

Comment:
This `libraryVariants.all` block is getting pretty complex. Can we pull this out somewhere shared to avoid the complexity needing to be in each project's build.gradle?